content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -333,7 +333,11 @@ using `allowedTopologies`.
 
 {{< note >}}
 `zone` and `zones` parameters are deprecated and replaced with
-[allowedTopologies](#allowed-topologies)
+[allowedTopologies](#allowed-topologies). When [GCE CSI Migration](
+https://kubernetes.io/docs/concepts/storage/volumes/#gce-csi-migration) is
+enabled, a GCE PD volume can be provisioned in a topology that does not match
+any nodes. With legacy pre-migration GCE PD, in this case an error will be produced
```

Comment:
maybe clarify that with csi migration, this will result in the pod failing to schedule.

Reply:
good idea, done.